src/expr/src/expr/expr_binary_nonnull.rs

```diff
@@ -206,6 +239,70 @@ macro_rules! gen_binary_expr_atm {
     };
 }
 
+/// `gen_binary_expr_bitwise` is similar to `gen_binary_expr_atm`.
+/// They are differentiate because bitwise operation does not support for float datatype.
```

Comment:
nit: `does not support for float datatype` -> `only supports integral datatype`.

numeric = integral + decimal + float

- numeric
  - exact
    - integral
      - smallint
      - integer
      - bigint
    - decimal
  - approx / float
    - real
    - double precision